applications/MappingApplication/custom_utilities/mapper_utilities.cpp

```diff
@@ -33,17 +34,15 @@ typedef std::size_t IndexType;
 void AssignInterfaceEquationIds(Communicator& rModelPartCommunicator)
 {
     const int num_nodes_local = rModelPartCommunicator.LocalMesh().NumberOfNodes();
-
     int num_nodes_accumulated = rModelPartCommunicator.GetDataCommunicator().ScanSum(num_nodes_local);
-
     const int start_equation_id = num_nodes_accumulated - num_nodes_local;
-
     const auto nodes_begin = rModelPartCommunicator.LocalMesh().NodesBegin();
 
-    #pragma omp parallel for
-    for (int i=0; i<num_nodes_local; ++i) {
-        ( nodes_begin + i )->SetValue(INTERFACE_EQUATION_ID, start_equation_id + i);
-    }
+    IndexPartition<unsigned int>(num_nodes_local).for_each(
+        [&](unsigned int i){
```

Comment:
I mean here you could do

[nodes_begin, start_equation_id](unsigned int i)

so that only those values are captured (and copied in this case)

Reply:
cool, thx for the suggestion